### PR TITLE
add SLE Micro 5.2 as Tech Prev

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://github.com/SUSE/spacewalk/issues/18608

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
[uyuni_client-configuration_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/9532338/uyuni_client-configuration_guide.pdf)

[suse_manager_client-configuration_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/9532349/suse_manager_client-configuration_guide.pdf)

**Comment by keichwa** [comment]
@mcalmer or @admd can you also please say whether we want to have this bckported to suma 4.2 (will suma 4.2 support sle micro 5.2?)

**Comment by keichwa** [comment]
> Not sure what you plan for 5.3. It is currently RC1 and should be GMC end of September. FCS is October 25th. SUMA 4.3.2 release is planned for October 17th (one week earlier)
> 
> Be careful: the channel name has changed in 5.3. It is now: sle-micro-5.3-pool- sle-micro-5.3-updates-

Thanks for the warning!  I could add 5.3 with a warning such as "when published" or 'when available''.  Or we can wait until 5.3 is actually out.

File: .changelog [hunk 1]
```diff
@@ -4,6 +4,8 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Added SLE Micro 5.2 as available as a technology preview in Client
+  Configuration Guide, and the IBM Z architectue for 5.1 and 5.2
```
Comment (keichwa) [inline .changelog:8]:
```suggestion
  Configuration Guide, and the IBM Z architecture for 5.1 and 5.2
```

**Review by 0rnela — APPROVED** [review]

**Review by mcalmer — APPROVED** [review]
ok, maybe you want to add 5.3 directly as well or via an extra PR

**Comment by keichwa** [comment]
> ok, maybe you want to add 5.3 directly as well or via an extra PR

ok, then I'll add it directly.

**Comment by keichwa** [comment]
I hope wording about 5.3 is ok.  We'll remove the release statement when 5.3 actually is available.  At least, this is the idea.

File: modules/client-configuration/pages/clients-slemicro.adoc [hunk 2]
```diff
@@ -83,11 +108,40 @@ The channels you need for this procedure are:
...
+| sle-micro-5.3-pool-arm64
+| sle-micro-5.3-updates-arm64
+
+| {micro} 5.2 {zsystems} ({s390x})
+| sle-micro-5.3-pool-s390x
+| sle-micro-5.3-updates-s390x
 
 |===
 
+{micro}{nbsp}5.3 is scheduled to be released end of October 2022.
```
Comment (mcalmer) [inline modules/client-configuration/pages/clients-slemicro.adoc:144]: Maybe just remove it.

File: modules/client-configuration/pages/clients-slemicro.adoc [hunk 3]
```diff
@@ -54,13 +57,35 @@ The products you need for this procedure are:
...
+
+| {micro} 5.3 {arm64}
+| SUSE Linux Enterprise Micro 5.3 aarch64
+
+| {micro} 5.3 {s390x}
+| SUSE Linux Enterprise Micro 5.3 s390x
 
 |===
 
+{micro}{nbsp}5.3 is scheduled to be released end of October 2022.
```
Comment (mcalmer) [inline modules/client-configuration/pages/clients-slemicro.adoc:88]: Maybe just remove it.

File: modules/client-configuration/pages/clients-slemicro.adoc [hunk 4]
```diff
@@ -3,8 +3,11 @@
 
 This section contains information about registering clients running these {nbsp}{micro} operating systems:
 
-* {micro}{nbsp}5.1 {x86_64}
-* {micro}{nbsp}5.1 {arm64}
+* {micro}{nbsp}5.1, 5.2, and 5.3 {x86_64}
+* {micro}{nbsp}5.1, 5.2, and 5.3 {arm64}
+* {micro}{nbsp}5.1, 5.2, and 5.3 {zsystems} (s390x)
+
+{micro}{nbsp}5.3 is scheduled to be released end of October 2022.
```
Comment (mcalmer) [inline modules/client-configuration/pages/clients-slemicro.adoc:10]: maybe just remove this line.

**Review by mcalmer — APPROVED** [review]
I would not say anything about the release date of 5.3.
Either remove it, or talk to Jiri if this working is ok.

But looks good

**Pull request merged**